input-output-hk/cardano-sl
# explorer-frontend: Update legal disclaimer

## Description

Updates the disclaimer text in the footer of https://cardanoexplorer.com/

## Linked issue

None

## Type of change

Translation strings.

## Screenshot

![explorer-footer](https://user-images.githubusercontent.com/1019641/70050603-1fb36500-161b-11ea-8bcd-0b7a341d745a.png)

## QA Steps

Run:

    nix-build -A explorerFrontend -o explorer-frontend
    cd explorer-frontend
    python3 -mhttp.server

Inspect the generated HTML. Open it in a web browser and check the footer text.
